examples/server_http.c

```diff
@@ -21,16 +21,11 @@
 
 static char *config_property = QUOTE(
     {
-        "transport": [
-            {
-                "value": "SCTP",
-                "precedence": 1
-            },
-            {
-                "value": "TCP",
-                "precedence": 1
-            }
-        ]
+      "transport": 
+          {
+              "value": ["TCP", "SCTP"],
+              "precedence": 1
+          }
```

Comment:
@bozakov on line 32, https is enabled, while config_property_https does not need the brackets apparently.